DirectEmployers/MyJobs
# [PD-1383]  dynamic reporting interface

This is a first cut at enough code to make a trivial report generate CSV. It uses some of the models present, not all yet.

Changes so far:

* Switch from table_name to app_name/model_name.
* Sample data for a contact records report.
  * So far only the model_name/app_name of the columns is used.
* A model to represent a report that exists.
* Some views to run a report and download csv, etc.

To test:

1. run manage.py loaddata myreports class_and_pres
2. Go to secure.my.jobs/reports/view/dynamicoverview
3. Enter 3 in configuration id, Run.
4, Click the report link to download csv.

What I expect to do before pushing:

* Use the configuration columns for query values.
* Build a trivial filter interface around configuration columns.

I marked XXX in a few spots in the code where I was unsure about design choices.
